src/main/java/org/motechproject/mots/domain/Community.java

```diff
@@ -26,6 +30,25 @@
   @Setter
   private Facility facility;
 
+  @Override
+  public boolean equals(Object o) {
```

Comment:
name is not unique it can cause problems later in hibernate, maybe you can make Facility required and name unique with combination with facility and add facility to equals and hash code

Reply:
makes sense